guides/controllers.md

```diff
@@ -351,48 +351,34 @@ Sometimes we need to communicate with users during the course of an action. Mayb
 
 The `Phoenix.Controller` module provides the [`put_flash/3`] to set flash messages as a key-value pair and placing them into a `@flash` assign in the connection. Let's set two flash messages in our `HelloWeb.PageController` to try this out.
 
-To do this we modify the `index` action as follows:
+To do this we modify the `home` action as follows:
 
 ```elixir
 defmodule HelloWeb.PageController do
   ...
-  def index(conn, _params) do
+  def home(conn, _params) do
     conn
-    |> put_flash(:info, "Welcome to Phoenix, from flash info!")
     |> put_flash(:error, "Let's pretend we have an error.")
-    |> render(:index)
+    |> render(:home, layout: false)
   end
 end
 ```
 
 In order to see our flash messages, we need to be able to retrieve them and display them in a template layout. We can do that using [`Phoenix.Flash.get/2`] which takes the flash data and the key we care about. It then returns the value for that key.
 
-For our convenience, the application layout, `lib/hello_web/components/layouts/app.html.heex`, already has markup for displaying flash messages.
+For our convenience, a `flash_group` component is already available and added to the beginning of our [welcome page]
 
 ```heex
-<.flash kind={:info} title="Success!" flash={@flash} />
-<.flash kind={:error} title="Error!" flash={@flash} />
-<.flash
-  id="disconnected"
-  kind={:error}
-  title="We can't find the internet"
-  close={false}
-  autoshow={false}
-  phx-disconnected={show("#disconnected")}
-  phx-connected={hide("#disconnected")}
->
-  Attempting to reconnect <.icon name="hero-arrow-path" class="ml-1 w-3 h-3 inline animate-spin" />
-</.flash>
+<.flash_group flash={@flash} />
 ```
 
-When we reload the [welcome page], our messages should appear just above "Welcome to Phoenix!"
+When we reload the [welcome page], our messages should appear in the top right corner of the page.
 
 The flash functionality is handy when mixed with redirects. Perhaps you want to redirect to a page with some extra information. If we reuse the redirect action from the previous section, we can do:
 
 ```elixir
-  def index(conn, _params) do
+  def home(conn, _params) do
     conn
-    |> put_flash(:info, "Welcome to Phoenix, from flash info!")
     |> put_flash(:error, "Let's pretend we have an error.")
     |> redirect(to: ~p"/redirect_test"))
   end
```

Comment:
Since you adjusted the example so that only one flash message is put, the following sentence should be adjusted as well:

"you will be redirected and the flash messages will be shown once more"
should become
"you will be redirected and the flash message will be shown once more"

Reply:
thanks! it should be good to go now.